### PR TITLE
Update icons.ui

### PR DESCRIPTION
**Pull request by zniavre**

### CODE REVIEW

**Comment by zniavre** [comment]
i hope i did well ...

File: pynicotine/gtkgui/ui/settings/icons.ui [hunk 1]
```diff
@@ -455,7 +455,7 @@
                   <object class="GtkLabel">
                     <property name="visible">True</property>
                     <property name="can_focus">False</property>
-                    <property name="label" translatable="yes">Hilite:</property>
+                    <property name="label" translatable="yes">Nick Hilite:</property>
```
Comment (mathiascode) [inline pynicotine/gtkgui/ui/settings/icons.ui:458]: This label is incorrect. The icon is also used whenever a new message is sent in a chat room.

**Comment by kiplingw** [comment]
@mathiascode, are you fine with this PR?

File: pynicotine/gtkgui/ui/settings/icons.ui (hunk 1 above)
Comment (zniavre) [reply pynicotine/gtkgui/ui/settings/icons.ui]: i revert the label to'Hilite" as it was before sorry for that ...

**Review by mathiascode — APPROVED** [review]

**Pull request merged**